examples/forall_multi-reductions.cpp

```diff
@@ -38,50 +36,50 @@ struct Backend
 
 auto example_policies = camp::make_tuple(
 
-      Backend<RAJA::seq_exec, RAJA::seq_multi_reduce>{"Sequential"}
+    Backend<RAJA::seq_exec, RAJA::seq_multi_reduce>{"Sequential"}
 
 #if defined(RAJA_ENABLE_OPENMP)
-    , Backend<RAJA::omp_parallel_for_exec, RAJA::omp_multi_reduce>{"OpenMP"}
+    ,
+    Backend<RAJA::omp_parallel_for_exec, RAJA::omp_multi_reduce>{"OpenMP"}
 #endif
 
 #if defined(RAJA_ENABLE_CUDA)
-    , Backend<RAJA::cuda_exec_async<256>, RAJA::cuda_multi_reduce_atomic>{"Cuda"}
+    ,
+    Backend<RAJA::cuda_exec_async<256>, RAJA::cuda_multi_reduce_atomic>{"Cuda"}
 #endif
 
 #if defined(RAJA_ENABLE_HIP)
-    , Backend<RAJA::hip_exec_async<256>, RAJA::hip_multi_reduce_atomic>{"Hip"}
+    ,
+    Backend<RAJA::hip_exec_async<256>, RAJA::hip_multi_reduce_atomic>{"Hip"}
 #endif
 
-    );
+);
 
-template < typename exec_policy, typename multi_reduce_policy >
+template <typename exec_policy, typename multi_reduce_policy>
 void example_code(RAJA::RangeSegment arange, int num_bins, int* bins, int* a)
 {
-  RAJA::MultiReduceSum<multi_reduce_policy, int>    multi_reduce_sum(num_bins);
-  RAJA::MultiReduceMin<multi_reduce_policy, int>    multi_reduce_min(num_bins);
-  RAJA::MultiReduceMax<multi_reduce_policy, int>    multi_reduce_max(num_bins);
+  RAJA::MultiReduceSum<multi_reduce_policy, int> multi_reduce_sum(num_bins);
```

Comment:
Here it did not change the indentation at the start of the line. I'm not seeing the pattern when it changes the indentation and when it does not.

Reply:
this should be fixed by `AlignConsecutiveDeclarations : true`